src/repository.rs

```diff
@@ -251,8 +252,59 @@ impl<M: Model> Repository<M> {
     /// ```
     pub async fn bulk_update<V, U>(&self, updates: V) -> Result<BulkUpdateResult>
     where
-        V: Borrow<Vec<U>>,
-        U: Borrow<BulkUpdate>,
+        M: Send + Sync,
+        V: Borrow<Vec<U>> + Send + Sync,
+        U: Borrow<BulkUpdate> + Send + Sync,
+    {
+        Ok(self.coll.bulk_update(&self.db, updates).await?)
+    }
+}
+
+
+#[async_trait]
+pub trait MongodmCollectionExt {
```

Comment:
[Convention](https://rust-lang.github.io/rfcs/0445-extension-trait-conventions.html#the-convention) is to call extension traits as `Type` + `Ext`:
```suggestion
pub trait CollectionExt {
```

(Or `CollectionBulkUpdateExt` since it only provides the `bulk_update` method)

Reply:
Fixed!